scripts/minishift.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "$0")"
+BASEDIR="$(pwd)"
+
+minishift delete -f
+
+MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --openshift-version v3.11.0 \
+--extra-clusterup-flags "--enable=*,service-catalog,automation-service-broker,template-service-broker"
+
+oc login -u system:admin
```

Comment:
it needs to specify the target host here. Otherwise this will try to login to an existing oc session (which could point to a different cluster).

Reply:
it should not be needed, when `minishift start` completes, oc configuration should target this newly created cluster

Reply:
ah, ok. The problem I have is that the minshift start command failed. So I think before this command, the script should fail if minishift failed to start?

Reply:
looks like it's ok, so don't worry about it.

Reply:
should be fixed now